Sources/NIOPerformanceTester/LockBenchmark.swift

```diff
@@ -17,7 +17,9 @@ import NIOConcurrencyHelpers
 import NIOCore
 import NIOPosix
 
-final class NIOLockBenchmark: Benchmark {
+final class NIOLockBenchmark: Benchmark, @unchecked Sendable {
+    // mutable state is protected by the lock
```

Comment:
Nit but couldn't we wrap `opsDone` in a `NIOLockedValueBox` instead and remove the `@unchecked`?

Reply:
From a `Sendable`-cleanness perspective, yes. However, this is a benchmark for `NIOLock`, so using `NIOLockedValueBox` would somewhat defeat the point of the benchmark!